hawkbit-rest/hawkbit-mgmt-api/src/main/java/org/eclipse/hawkbit/mgmt/rest/api/MgmtDistributionSetRestApi.java

```diff
@@ -757,7 +757,7 @@ ResponseEntity<List<MgmtMetadata>> createMetadata(@PathVariable("distributionSet
         always be a list of software module IDs. Required permissions: READ_REPOSITORY and UPDATE_REPOSITORY
         """)
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "200", description = "Successfully retrieved"),
+        @ApiResponse(responseCode = "201", description = "Successfully created"),
```

Comment:
Assigning a SM to a DS also returns status 200 in the implementation.

Reply:
I think you got my point here - the other changes also should be double checked with implementation logic.